modules/core/src/main/scala/org/tesselation/domain/gossip/RumorStorage.scala

```diff
@@ -13,4 +14,8 @@ trait RumorStorage[F[_]] {
 
   def getSeenHashes: F[List[Hash]]
 
+  def tryUpdateCounter(rumor: Rumor): F[Option[Long]]
```

Comment:
I think it should just return `F[Boolean]`

Reply:
`tryGetAndUpdateCounter` then?

Reply:
Well if the method will return `Boolean` instead of an `Option[Long]` the `Get` verb doesn't fit, so I would stay with the old name.

Reply:
No, I need an `Option[Long]`. Same as original method `getAndUpdate` does